### DISCUSSION TITLE
Changing password

### DISCUSSION THREAD

**Discussion by andriusign** [directus/directus#4027] [Archive]
Directus documentation of password change API says it has to be called with GET method.
![Screenshot from 2021-02-12 15-46-24](https://user-images.githubusercontent.com/14263959/107776182-ea8bc200-6d49-11eb-86c5-f93e80d40780.png)

However calling it with GET method does not change password.
Using POST method I get the following error:

```
{
    "errors": [
        {
            "message": "Route /auth/oauth doesn't exist.",
            "extensions": {
                "code": "ROUTE_NOT_FOUND"
            }
        }
    ]
}
```

What is the proper way to change password?

**Comment by rijkvanzanten** [comment]
Good catch, i've updated it in 6baea6a69d55c07168c667e4f389d945d7480ebf
